<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>420</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>440</width>
    <height>420</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MasterMind</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeHistory">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>421</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>No.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Guess</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Output</string>
     </property>
    </column>
   </widget>
   <widget class="QLCDNumber" name="lcdLives">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textChecked">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLives">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Lives</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGuess">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Guess!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineGuess">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>258</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_game"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuCheats">
    <property name="title">
     <string>Cheats</string>
    </property>
    <addaction name="actionShow_Goal"/>
    <addaction name="actionInfinate_Lives"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGuide"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_MasterMind_Qt"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCheats"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionNew_game">
   <property name="text">
    <string>New game</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionShow_Goal">
   <property name="text">
    <string>Show Goal</string>
   </property>
  </action>
  <action name="actionInfinate_Lives">
   <property name="text">
    <string>HESOYAM (inf lives)</string>
   </property>
  </action>
  <action name="actionGuide">
   <property name="text">
    <string>Guide</string>
   </property>
  </action>
  <action name="actionAbout_MasterMind_Qt">
   <property name="text">
    <string>About MasterMind-Qt</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionAbout_Qt</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_aboutqt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout_MasterMind_Qt</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_Goal</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_showgoal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonGuess</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_guess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInfinate_Lives</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_inflives()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_game</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_newgame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGuide</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_guide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineGuess</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>action_guess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>action_aboutqt()</slot>
  <slot>action_about()</slot>
  <slot>action_guide()</slot>
  <slot>action_showgoal()</slot>
  <slot>action_guess()</slot>
  <slot>action_inflives()</slot>
  <slot>action_newgame()</slot>
 </slots>
</ui>
